backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/llvm/IrToBitcode.kt

```diff
@@ -130,11 +131,48 @@ internal fun emitLLVM(context: Context) {
         irModule.acceptVoid(codegenVisitor)
     }
 
+    when (context.shouldEmbedBitcode()) {
+        EmbedBitcode.ON -> embedBitcode(context, marker = false)
+        EmbedBitcode.MARKER -> embedBitcode(context, marker = true)
+        EmbedBitcode.OFF -> {}
+    }
+
     if (context.shouldContainDebugInfo()) {
         DIFinalize(context.debugInfo.builder)
     }
 }
 
+private fun embedBitcode(context: Context, marker: Boolean) {
+    val isIos = context.config.platform.configurables.target.family == Family.IOS
```

Comment:
Hmm, why IOS is specific here?

Reply:
Taken from https://github.com/rust-lang/rust/pull/48896/commits/0e0f74bcf93341ff3b1d8b59f3639416a159a140#diff-a3b24dbe2ea7c1981f9ac79f9745f40aR851 and https://github.com/llvm-mirror/clang/blob/master/lib/CodeGen/BackendUtil.cpp#L1235
the section name is just different on that platform. Unsure if is should be on macos as well?